encrypted-media-respec.html

```diff
@@ -2816,7 +2816,7 @@
         <section>
           <h4><a>MediaEncryptedEventInit</a></h4>
           <div><pre class="idl">dictionary MediaEncryptedEventInit : EventInit {
-             DOMString    initDataType = "";
+             required DOMString    initDataType
```

Comment:
I agree. I guess it wasn't clear what to do about the nullable initData, especially given the discussion in the issue.

Should this be `required ArrayBuffer? initData`?

Is the following valid in that case?
```
{ initDataType = "",
  initData = null; }
```